### PR TITLE
[Portal] Update user detail account security screen

### LINKED ISSUE #330: Update user detail account security screen and add reset password + verification code screens
Account security tab:

- [x]  one sub section per authenticator type
- [x] show detail info in sub section with remove button
- [x] for password, show `Reset password` button
- [x] `Reset password` link to reset password page
- [x] dummy `Remove` button otherwise
- [x] update icon, one icon for each kind of claim
  - email
  - phone
  - icon using fluentUI
- [x] add authentication button
  - [x] context menu, item is link to verification code screens and add password screen

Reset password page
- [x] setup reset password page
- [x] same password field as add user page

Verification code screen
- [x] setup add verification code page

Add password page
- [x] reuse form from reset password page

### PR DESCRIPTION
**Pull request by axolotl0212**
Closes #330 

- update layout
  - remove detail button
  - display all authenticators, one list per authenticator type
- add remove button for secondary authenticator
- create reset password button

### CODE REVIEW

**Comment by axolotl0212** [comment]
removed add second authenticator pages

**Comment by louischan-oursky** [comment]
Conflicted.

**Comment by axolotl0212** [comment]
@louischan-oursky  rebased

File: portal/src/graphql/adminapi/UserDetailsAccountSecurity.tsx [hunk 1]
```diff
@@ -63,58 +96,278 @@ function getLocaleKeyWithAuthenticatorType(
...
+    lastUpdated,
+  };
+}
+
+function getTotpDisplayName(
+  totpAuthenticatorClaims: Record<string, unknown>
+): string {
+  for (const [key, claim] of Object.entries(totpAuthenticatorClaims)) {
+    // expected form: https://authgear.com/claims/totp/display_name
+    const lastSegment = key.split("/").pop();
```
Comment (louischan-oursky) [inline portal/src/graphql/adminapi/UserDetailsAccountSecurity.tsx:118]: You just need to perform a simple string comparison here. The value `https://authgear.com/claims/totp/display_name` is static.

File: portal/src/graphql/adminapi/ResetPasswordScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        onChange={onNewPasswordChange}
+        passwordPolicy={passwordPolicy}
+      />
+      <TextField
+        className={cn(styles.passwordField, styles.confirmPasswordField)}
+        label={renderToString("ResetPasswordScreen.confirm-password")}
+        value={confirmPassword}
+        onChange={onConfirmPasswordChange}
+        errorMessage={confirmPasswordErrorMessage}
+      />
```
Comment (louischan-oursky) [inline portal/src/graphql/adminapi/ResetPasswordScreen.tsx:156]: The type should also be `password`, not plaintext.

Comment (axolotl0212) [reply portal/src/graphql/adminapi/ResetPasswordScreen.tsx:157]: Thanks, forgotten to put type here

**Comment by axolotl0212** [comment]
@louischan-oursky updated

**Pull request merged**